magento/magento2
# Add possibility to disable the admin notification window and toolbar

### Description (*)
Add possibility to disable the admin notification window (the popup about Magento 2.X release after logging in) and toolbar (the notification icon with message count in the header).

### Fixed Issues (if relevant)
None that I know of.

### Manual testing scenarios (*)

1. Have items in your admin notification inbox and make sure you see the notification window after logging in.
2. Set configuration **Show notification window after logging in** to **No**. 
3. Log out and log back in. The notification window should not appear.
4. Set configuration **Show notifications in toolbar** to **No**.
5. The notification icon in the header should not appear.

### Questions or comments
This is kind of a short feature request/addition to the current feature set. The Magento 2 notifications can be relevant to merchants, but in many situations they are prone to be 'spammy' and 'irrelevant'. It should be possible to disable these notifications showing up.

### Contribution checklist (*)
